sisl/orbital.py

```diff
@@ -1130,3 +1130,50 @@ def __setstate__(self, d):
             self.__init__(d["name"], q0=d["q0"], tag=d["tag"])
         else:
             self.__init__(d["name"], (d["r"], d["f"]), q0=d["q0"], tag=d["tag"])
+
+
+@set_module("sisl")
+class HydrogenicOrbital(AtomicOrbital):
+    r""" A hydrogen-like atomic orbital defined by an effective atomic number Z
```

Comment:
The first line should be a single line (shorthand in documentation).

Then additional descriptions should follow.

I think it would be nice to add the following information:
```
A hydrogenic atom (Hydrogen-like) is an atom with a single valence electron.
```

Could we also put:

```
See [1]_ for details about the hydrogenic atom.

References
---------------
.. [1] : https://en.wikipedia.org/wiki/Hydrogen-like_atom
```
this will put the link in a reference section.

Reply:
Better now?